pipeline-scripts/uperf.groovy

```diff
@@ -24,30 +24,69 @@ stage ('uperf') {
 			def orchestration_user = uperf_properties['ORCHESTRATION_USER']
 			def orchestration_host = uperf_properties['ORCHESTRATION_HOST']
 			def sshkey_token = uperf_properties['SSHKEY_TOKEN']
+			def es_user = uperf_properties['ES_USER']
+			def es_password = uperf_properties['ES_PASSWORD']
 			def es_server = uperf_properties['ES_SERVER']
 			def es_port = uperf_properties['ES_PORT']
 			def metadata_collection = uperf_properties['METADATA_COLLECTION']
 			def baseline_uperf_uuid = uperf_properties['BASELINE_UPERF_UUID']
 			def hostnetwork_test = uperf_properties['HOSTNETWORK_TEST']
 			def pod_test = uperf_properties['POD_TEST']
 			def service_test = uperf_properties['SERVICE_TEST']
-			def cerberus_url = uperf_properties['CERBERUS_URL']			
-			
+			def smoke_test = uperf_properties['SMOKE_TEST']
+			def compare = uperf_properties['COMPARE']
+			def baseline_cloud_name = uperf_properties['BASELINE_CLOUD_NAME']
+			def es_user_baseline = uperf_properties['ES_USER_BASELINE']
+			def es_password_baseline = uperf_properties['ES_PASSWORD_BASELINE']
+			def es_server_baseline = uperf_properties['ES_SERVER_BASELINE']
+			def es_port_baseline = uperf_properties['ES_PORT_BASELINE']
+			def baseline_hostnet_uuid = uperf_properties['BASELINE_HOSTNET_UUID']
```

Comment:
Do we want to just encapsulate this into a python script vs passing the uuids?

Reply:
yes, working on it but till then can we use this to get 3-4 runs against 4.5 readout uuids.
Short term PR to get us rolling

Reply:
and even after that I feel we need to have this just in case we are not comparing it against a gold run but against some normal or or even cross-platform
We can keep the values to these uuids empty and only fill in when we need it.

Reply:
ack